vault/logical_system_paths.go

```diff
@@ -815,7 +905,19 @@ func (b *SystemBackend) pluginsCatalogListPaths() []*framework.Path {
 			Operations: map[logical.Operation]framework.OperationHandler{
 				logical.ListOperation: &framework.PathOperation{
 					Callback: b.handlePluginCatalogTypedList,
-					Summary:  "List the plugins in the catalog.",
+					Responses: map[int][]framework.Response{
+						http.StatusOK: {{
+							Description: "OK",
+							Fields: map[string]*framework.FieldSchema{
+								"keys": {
+									Type:        framework.TypeStringSlice,
+									Description: "List of plugin names in the catalog",
+									Required:    true,
```

Comment:
```suggestion
									Required:    false,
```
`logical.ListResponse` only has `keys` when there's more than one item
https://github.com/hashicorp/vault/blob/2267b042dba5ebe278867662ea059c4bda17356d/sdk/logical/response.go#L140-L142

_but_ would think we always have at least 1 plugin... 🤷

Reply:
Ah good catch.